spec/unit/indirector/file_bucket_file/file_spec.rb

```diff
@@ -93,6 +93,7 @@ def save_bucket_file(contents, path = "/who_cares")
       describe "when supplying a path" do
         with_digest_algorithms do
             it "should store the path if not already stored" do
+              pending("Windows Long File Name support is incomplete PUP-8257") if Puppet.features.microsoft_windows? && digest_algorithm == 'sha384'
               if Puppet::Util::Platform.windows? && ("#{digest_algorithm}" == "sha512")
```

Comment:
So I see the existing skips a line below... if we're going to continue these checks, how about we update the conditionals below instead like:

```ruby
if Puppet::Util::Platform.windows? && (['sha512', 'sha384'].include? digest_algorithm)
  pending "PUP-8257: Skipping file bucket test on windows for #{digest_algorithm} due to long path names"
```